numba/tests/test_array_manipulation.py

```diff
@@ -408,11 +408,13 @@ def check_all_axes(arr):
         # 1d
         arr = np.arange(5)
         check_all_axes(arr)
+        check(arr, (1, 2))
         # 3d (C, F, A)
         arr = np.arange(24).reshape((2, 3, 4))
         check_all_axes(arr)
         check_all_axes(arr.T)
         check_all_axes(arr[::-1])
+        check(arr, (1, 3))
```

Comment:
It would be good to include:
* tests ensuring that repeated axis values raise `ValueError('repeated axis')`.
* tests for out of bound axis handling
* tests for negative and zero axes (and e.g. wrap around hits the repeated axis value exception)
* tests for a large number of axes e.g. `(0, 1, 2, 3, 4, 5, 6, 7, 8)`
* tests for empty tuple e.g. `np.expand_dims(np.arange(5), ())`
* tests involving strided arrays
* tests involving 0d arrays.

Reply:
Added more tests in [60d59ab](https://github.com/numba/numba/pull/7604/commits/60d59abc7cdf0b3654604f24b2024dca7bf2043c)